docs/SUPPORTED_FORMATS.md

```diff
@@ -0,0 +1,51 @@
+### RGB-like
+
+| Name    |     VL42 Identifier    |   current support  |   future support   |
```

Comment:
is the `future support` column still needed?

Reply:
column removed